### PR TITLE
update `ol` to 6.15.1

### PR DESCRIPTION
**Pull request by olliethomas**
changed a couple of imports, and removed some that weren't being used.

### CODE REVIEW

**Comment by olliethomas** [comment]
Not really sure how to test this....

**Comment by olliethomas** [comment]
Closes #22

**Comment by robinjam** [comment]
Would it be possible to change back the formatting? It's hard to see what's going on with all the semicolons getting added everywhere and all the quotes changing

**Comment by olliethomas** [comment]
> Would it be possible to change back the formatting? It's hard to see what's going on with all the semicolons getting added everywhere and all the quotes changing

balls... I had run `prettier` on file save

**Comment by robinjam** [comment]
Incidentally we should set up some kind of code formatting at some point, maybe there should be a Trello card for that

I strongly object to having semicolons on every line but making the quotes consistent would be good

**Comment by olliethomas** [comment]
yeah... I am a big fan of code formatters. I don't really have opinions on formatting other than it should be consistent.

File: app/javascript/modelling/components/InputOverlayComponent.js [hunk 1]
```diff
@@ -1,7 +1,3 @@
-import { createEmpty as createEmptyExtent, extend as extendExtent } from 'ol/extent'
-import GeoJSON from 'ol/format/GeoJSON'
-import GeometryType from 'ol/geom/GeometryType'
-import { createXYZ } from 'ol/tilegrid'
```
Comment (olliethomas) [inline app/javascript/modelling/components/InputOverlayComponent.js:4]: these imports were unused

File: app/javascript/modelling/worker/rasteriseOverlay.js [hunk 2]
```diff
@@ -1,7 +1,5 @@
 import { createEmpty as createEmptyExtent, extend as extendExtent } from 'ol/extent'
 import GeoJSON from 'ol/format/GeoJSON'
-import GeometryType from 'ol/geom/GeometryType'
```
Comment (olliethomas) [inline app/javascript/modelling/worker/rasteriseOverlay.js:3]: GeometryType is now removed. I can't really tell, but I think you're meant to use strings comparisons now (which seems bad to me, but what do I know)
https://openlayers.org/en/latest/apidoc/module-ol_geom_Geometry.html#~Type

File: app/javascript/modelling/worker/rasteriseOverlay.js [hunk 3]
```diff
@@ -1,7 +1,5 @@
...
-import Polygon from 'ol/geom/Polygon'
```
Comment (olliethomas) [inline app/javascript/modelling/worker/rasteriseOverlay.js:4]: removed as wasn't used.

File: app/javascript/react/components/Map.jsx [hunk 4]
```diff
@@ -5,7 +5,10 @@ import { asArray } from 'ol/color'
 import { defaults as defaultControls, ZoomToExtent } from 'ol/control'
 import { GeoJSON } from 'ol/format'
 import { Image as olImageLayer, Tile as olTileLayer, Vector as olVectorLayer } from 'ol/layer'
-import { ImageStatic, OSM, Vector as VectorSource, XYZ } from 'ol/source'
+import Static from 'ol/source/ImageStatic';
+import VectorSource from 'ol/source/Vector';
+import OSM from 'ol/source/OSM';
+import XYZ from 'ol/source/XYZ'
```
Comment (olliethomas) [inline app/javascript/react/components/Map.jsx:11]: Importing from `source` gave an error about could import getFloat16 from `petamoriken/float16`

**Comment by olliethomas** [comment]
As in #30 - I'm really not sure why the Gemfile.lock has been updated.

**Pull request merged**